code-studio/src/js/components/share_dialog_body.jsx

```diff
@@ -92,6 +92,16 @@ var ShareDialogBody = React.createClass({
         appType={this.props.appType}/>;
     }
 
+    var exportButton = null;
+    if (this.props.appType === 'applab') {
+      // TODO: Can I not proliferate the use of global references to Applab somehow?
```

Comment:
I guess I can move the dependency on the global higher up in the stack to where ShareDialog gets used, but it's still going to be a global :(